src/rpc/blockchain.cpp

```diff
@@ -2293,15 +2297,20 @@ UniValue scantxoutset(const JSONRPCRequest& request)
         g_scan_progress = 0;
         int64_t count = 0;
         std::unique_ptr<CCoinsViewCursor> pcursor;
+        CBlockIndex* tip;
         {
```

Comment:
Can you remove unused curly braces

Reply:
The block is required for tcs_main scoped lock.

Reply:
Can you show me this documentation? I'm not familiar with it

Reply:
You cannot access variables which are shared across threads without grabbing the correct lock.